guides/rif-relay/smart-wallets.md

```diff
@@ -5,14 +5,16 @@ tags: rif, envelope, relay, user, guide, smart, wallet
 permalink: /guides/rif-relay/smart-wallets/
 ---
 
+This guide is intended for testing purposes and we will be using additional testing contracts that were included in the project, like the TestToken(ERC20). All the utils scripts are executed from the account[0] from the regtest blockchain. 
```

Comment:
I don't think this is true for the whole guide. I think that the SW deployment and usage is the same regardless of the network we use. Unless I'm missing something, just the `TestToken` usage is for testing purposes only.

Reply:
updated